CoEpi/repo/AlertRepo.swift

```diff
@@ -1,35 +1,64 @@
 import Foundation
 import RxCocoa
 import RxSwift
+import os.log
 
 protocol AlertRepo {
```

Comment:
"Alert" and "Report" are used as synonyms. We should standardize on one (I prefer report).

Reply:
I actually changed it in Android to be `Alert`, to differentiate better from outgoing reports.

Reply:
I actually changed it in Android to be `Alert`, which is an incoming matched report. This differentiates it from non matching reports and outgoing reports.